pkg/pillar/types/clustertypes.go

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) 2024 Zededa, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package types
+
+import (
+	"net"
+
+	uuid "github.com/satori/go.uuid"
+)
+
+const (
+	ClusterStatusPort = "12346" // ClusterStatusPort - Port for k3s server for cluster status advertise
+)
+
+// EdgeNodeClusterConfig - Configuration for cluster multi-node from controller
+type EdgeNodeClusterConfig struct {
+	ClusterName      string
+	ClusterID        UUIDandVersion
+	ClusterInterface string
```

Comment:
Add comment that this is a logical label of the port.

Reply:
OK.